lib/hypernova/controller_helpers.rb

```diff
@@ -210,7 +210,11 @@ def render_react_component_without_response(component, data = {})
 
       new_body.scan(RENDER_STATUS_REGEX) do |matched|
         if matched
-          render_result_without_cache
+          if controller.respond_to?(:response)
```

Comment:
why do we need to add this condition?

Reply:
Because the method "render_result_without_cache" only valid for controller. If ActionMailer call the method it can't respond to response and will report an error.